ghost.py

```diff
@@ -453,10 +523,11 @@ def _prettify_list(items):
     token_normalize_func=lambda param: param.lower())
 
 # Currently static. We'll see how backend implementations go and adjust
-# to dymanic mapping accordingly
+# to dynamic mapping accordingly
 STORAGE_NAME_MAPPING = {
     'tinydb': TinyDBStorage,
     'sqlalchemy': SQLAlchemyStorage,
+    'consul': ConsulStorage
```

Comment:
all the other parameters have to do with authentication, which is optional in consul. It is possible to only secure consul by means of a firewall/secgroup, allowing all requests (btw, this is what we do in the HA poc). In this case, the url is enough.

(well, the `directory` parameter doesn't have to do with auth - it's a common prefix for your keys, but there isn't many usecases where you'd want to change that - i think only if you wanted to have several separate ghost installs on the one consul cluster)